producers/service.go

```diff
@@ -23,14 +23,14 @@ type kubernetesServiceProducer struct {
 }
 
 func NewKubernetesService() (*kubernetesServiceProducer, error) {
-	client, err := kubernetes.NewClient(params.kubeServer)
+	client, err := kubernetes.NewClient(kubernetesParams.kubeServer)
```

Comment:
seems managing flags is a bit complicated in this place, service/ingress flags are populated in the kubernetes file. To solve this problem we probably need to resort to another flag lib.